modules/openapi-generator/src/main/resources/cpp-ue4/api-operations-source.mustache

```diff
@@ -86,7 +81,7 @@ FString {{classname}}::{{operationIdCamelCase}}Request::ComputePath() const
 	{{/pathParams.0}}
 	{{#pathParams.0}}
 	TMap<FString, FStringFormatArg> PathParams = { {{#pathParams}}
-	{ TEXT("{{baseName}}"), ToStringFormatArg({{paramName}}) }{{^-last}},{{/-last}}{{/pathParams}} };
+	{ TEXT("{{baseName}}"), FStringFormatArg(ToUrlString({{paramName}})) }{{^-last}},{{/-last}}{{/pathParams}} };
```

Comment:
Yes, you're right. `ToString` has default to use `FStringFormatArg`, which leads to double formatting. While surfing the code, I found that it's better to use Unreal's `LexToString` instead of `FString::Format` as a default implementation of `ToString`. I will prepare for it.